android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java

```diff
@@ -415,7 +414,12 @@ public void call_sendMessage(String uuid, String content, String contentType, St
 
     final CallMessage callMessage = new CallMessage.Builder(_messageType)
       .contentType(_contentType).content(content).build();
-    promise.resolve(callRecord.getVoiceCall().sendMessage(callMessage));
+
+    if (CallRecord.CallInviteState.ACTIVE == callRecord.getCallInviteState()) {
```

Comment:
can we clean this up?
maybe make it 
 promise.resolve((CallRecord.CallInviteState.ACTIVE == callRecord.getCallInviteState()) ? callRecord.getCallInvite().sendMessage(callMessage) : callRecord.getVoiceCall().sendMessage(callMessage));